src/components/textSelection/TextSelectionMenu.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import './textSelectionMenu.scss';
+import comment from '../../images/comment.svg'
+import git from '../../images/git.svg'
+
+const TextSelectionMenu = ({ options, showDialog }) => {
+  const [style, setStyle] = useState({
+    visibility: 'hidden',
+    zIndex: -100,
+  });
+
+
+  useEffect(() => {
+    setStyle(options)
+  }, [options]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
why need eslint-disable-line react-hooks/exhaustive-deps?

Reply:
cause eslint keeps prompting me to add another dependency that I don’t need